components/layout/Drawer/Drawer.tsx

```diff
@@ -20,6 +22,7 @@ const Drawer = ({
   const uidLabel = useUID();
   const uidDesc = useUID();
   const { t } = useTranslation();
+  const loading = useQueryStateContext().loading;
```

Comment:
This is a little bit cleaner way of saying the same thing:
`const { loading } = useQueryStateContext();`